.e2eignore

```diff
@@ -0,0 +1,9 @@
+docs/**
```

Comment:
please add [release_versions](https://github.com/percona/percona-server-mongodb-operator/blob/main/e2e-tests/release_versions) to the list.